### PR TITLE
Design fixes

### PR DESCRIPTION
**Pull request by thasnim-cm**
- Quick links and searchbar alignment in navbar.
- Quick link cards alignment fix
- Disabled future time in custom calendar.
- Added pst date in calendar filter in opm, opm comparison and sessions in the initial render
- Appcontent alignment fix.
- Date format of filteredcards changed to mm/dd/yyyy.
- PST conversion twice issue in OPM Comparison fix.
- Added time in calendar as current pst date minus 10 mins in opm and sessions initial render.

### CODE REVIEW

File: src/components/common/CustomCalendar.tsx [hunk 1]
```diff
@@ -109,25 +123,33 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
         />
         )
         } />
+      <CustomToast
+        onHide={() => setShowFutureDateToast(false)}
+        showToast={showFutureDateToast}
+        severity="warn"
+        detail="Please select a date and time on or before the current moment."
```
Comment (raj-kumar-litmus) [inline src/components/common/CustomCalendar.tsx]: Please don't select a future date and time.

File: src/components/common/CustomCalendar.tsx [hunk 2]
```diff
@@ -109,25 +123,33 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
...
+        position="top-center"
+        life={3000}
```
Comment (raj-kumar-litmus) [inline src/components/common/CustomCalendar.tsx]: Don't keep life.
Let the toast be open until user manually closes it.
It's important that the user doesn't miss the toast.
So let the user themselves close the toast.

File: src/components/common/CustomCalendar.tsx [hunk 3]
```diff
@@ -109,25 +123,33 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
...
     </div>
   )
 }
 
 const ManualInputTime: FC<ManualInputTimeProps> = ({ hour, minute, handleHourChange, handleMinuteChange, ampm, toggleAmPmChange }) => {
   return (
     <div className="flex justify-evenly w-[12rem] mx-auto items-center">
-      <InputNumber value={hour} step={1} onValueChange={(e) => handleHourChange(e)} showButtons buttonLayout="vertical" min={0} max={12} /> :
-      <InputNumber value={minute} onValueChange={(e) => handleMinuteChange(e)} showButtons buttonLayout="vertical" step={1} min={0} max={59} />:
+      <CustomInputNumber value={hour < 1 ? 12 : hour} step={1} onValueChange={(e) => handleHourChange(e)} showButtons buttonLayout="vertical" min={0} max={12} prefix={hour && hour < 10 && "0"} /> :
```
Comment (raj-kumar-litmus) [inline src/components/common/CustomCalendar.tsx]: Such a long line.
On save, it should have aligned into multiple lines.
Can you check if your linter is working as expected ?

Comment (thasnim-cm) [reply src/components/common/CustomCalendar.tsx]: I checked prettier settings. I couldn't find a solution. It wont automatically update for me. For now, I  fixed this

File: src/components/common/CustomCalendar.tsx (hunk 2 above)
Comment (thasnim-cm) [reply src/components/common/CustomCalendar.tsx]: Removed life

File: src/components/common/CustomCalendar.tsx (hunk 1 above)
Comment (thasnim-cm) [reply src/components/common/CustomCalendar.tsx]: Done

File: src/components/quicklinks.tsx [hunk 4]
```diff
@@ -74,22 +89,22 @@ const QuickLinks = () => {
...
-          className="custom-btn block"
-          />
-        }
       </div>
-    </div>
+      {showSeeMoreBtn &&
+        <CustomButton
+          label="See More"
+          onClick={onSeeMoreClick}
+          className="custom-btn text-center quick-link-btn border-white bg-[#161A1D]"
```
Comment (AdithyaShaj) [inline src/components/quicklinks.tsx]: replace hex color code with corresponding classname

**Review by AdithyaShaj — APPROVED** [review]

**Review by sarath-sasi-panickaveetil — APPROVED** [review]
Looks good to me

**Pull request merged**